Makefile

```diff
@@ -53,6 +57,11 @@ update-manifests:
 	sed 's#\(image:\) .*#\1 $(IMAGE)#' manifests/deis-workflow-rc.yml \
 		> manifests/deis-workflow-rc.tmp.yml
 
+deploy: docker-build docker-push
+	sed 's#\(image:\) .*#\1 $(IMAGE)#' /tmp/deis-$(COMPONENT) | kubectl apply --validate=true -f -
+	kubectl scale rc deis-workflow --replicas 0 --namespace deis
+	kubectl scale rc deis-workflow --replicas $(DESIRED_REPLICAS) --namespace deis
+
```

Comment:
You're pooping `deis-$(COMPONENT)` files in my temp directory, @helgi. :laughing: